package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "daffodil-debugger",
+  "name": "apache-daffodil-vscode",
   "displayName": "Apache Daffodil Schema Debugger",
```

Comment:
Should this name be Apache Daffodil VS Code Extension? Or is this display name for something else?

Reply:
It could be if that would be more desired

Reply:
Yeah, seems commit b5185aa26fefd0c39b47957e84467959875dc348 updated most places to use the "Apache Dafodil VS Code Extension" name. Probably just missed this one.